Studies/source/VariableDistribution.cxx

```diff
@@ -88,26 +87,11 @@ class VariablesDistribution {
         {
             if ( entry.channel != "" && args.tree_name() != entry.channel ) continue;
             auto input_file = root_ext::OpenRootFile(args.input_path()+"/"+entry.filename);
-            EventTuple tuple(args.tree_name(), input_file.get(), true, {} , GetMvaBranches());
-            Long64_t tot_entries = 0;
-            for (Long64_t  current_entry = 0; current_entry < tuple.GetEntries(); current_entry++) {
-                tuple.GetEntry(current_entry);
-                const Event& event = tuple.data();
-
-                if (static_cast<EventEnergyScale>(event.eventEnergyScale) != EventEnergyScale::Central || (event.q_1+event.q_2) != 0 || event.jets_p4.size() < 2
-                    || event.extraelec_veto == true || event.extramuon_veto == true || event.jets_p4[0].eta() > cuts::btag_2016::eta
-                    || event.jets_p4[1].eta() > cuts::btag_2016::eta )
-                    continue;
-
-                LorentzVectorE_Float bb = event.jets_p4[0] + event.jets_p4[1];
-
-                if (!cuts::hh_bbtautau_2016::hh_tag::IsInsideEllipse(event.SVfit_p4.mass(), bb.mass()))
-                    continue;
-
-                tot_entries++;
+            auto tuple = ntuple::CreateEventTuple(args.tree_name(), input_file.get(), true, ntuple::TreeState::Skimmed);
```

Comment:
For VariablesDistribution.cxx, better to add an option if use skimmed tuple or the full one. Like this we can used to study if some cuts (e.g. mass window) can be relaxed.